Modules/Scripted/DICOMLib/Widgets/Resources/UI/qSlicerDICOMExportDialog.ui

```diff
@@ -255,7 +255,7 @@ If unchecked, the exported dataset will be added to the DICOM database.</string>
          <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Drives|ctkPathLineEdit::Executable|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::PermissionMask|ctkPathLineEdit::Readable|ctkPathLineEdit::Writable</set>
         </property>
         <property name="options">
-         <set>ctkPathLineEdit::ShowDirsOnly</set>
+         <set>QFileDialog::ShowDirsOnly</set>
```

Comment:
@jcfr If you don't want this change, you can revert it

Reply:
The proposed change makes sense, I suggest we apply it once we have addressed:
* https://github.com/commontk/CTK/issues/1129

Reply:
> I figured it was such a minor compatibility change, but I can work to add it as a deprecation on the CTK side.

I will do a search to assess if the this is used on the C++ side of Slicer extensions.

The issues is that the `ctkDirectoryButton::Option` and `ctkDirectoryButton::Option` enum values are not available anymore from the class as they derive from `QWidget` and not from `QFileDialog`.